src/apps/menu/menu.tsx

```diff
@@ -0,0 +1,207 @@
+        <div className="modal__screen-reader-description" id="describemodal" />
+        <button
+          type="button"
+          aria-describedby="describemodal"
+          className="btn-ui modal-btn-close"
+          aria-label="close modal"
+        >
+          <img src={CloseIcon} alt="close modal button" />
+        </button>
+        <div className="modal-header">
+          <div className="modal-header__avatar">
+            <div className="avatar bg-global-secondary">
+              <img src={profileIcon} alt="" />
+            </div>
+          </div>
+          <div className="modal-header__name text-header-h4">
+            {userData?.patron?.name}
+          </div>
+          <Link
+            href={menuViewYourProfileTextUrl}
+            className="link-tag modal-header__link color-secondary-gray"
+          >
+            {t("menuViewYourProfileText")}
+          </Link>
+        </div>
+        <div className="modal-profile__notifications">
+          {loansOverdue !== 0 && (
+            <MenuNotification
+              notificationNumber={loansOverdue}
+              notificationText={t("menuNotificationLoansExpiredText")}
+              notificationColor="danger"
+              notificationLink={menuNotificationLoansExpiredUrl}
+            />
+          )}
+          {loansSoonOverdue !== 0 && (
+            <MenuNotification
+              notificationNumber={loansSoonOverdue}
+              notificationText={t("menuNotificationLoansExpiringSoonText")}
+              notificationColor="warning"
+              notificationLink={menuNotificationLoansExpiringSoonUrl}
+            />
+          )}
+          {reservationsReadyForPickup !== 0 && (
+            <MenuNotification
+              notificationNumber={reservationsReadyForPickup}
+              notificationText={t("menuNotificationReadyForPickupText")}
+              notificationColor="info"
+              notificationLink={menuNotificationReadyForPickupUrl}
+            />
+          )}
+        </div>
+        <div className="modal-profile__container">
+          <div className="modal-profile__links">
+            <div className="link-filters">
+              <MenuNavigationList
+                menuNavigationData={menuNavigationData}
+                loansCount={loansCount}
+                reservationCount={reservationCount}
+                feeCount={feeCount}
+              />
+            </div>
+          </div>
+          <div className="modal-profile__btn-logout">
+            <Link href={menuLogOutUrl}>
+              <button
+                type="button"
+                className="btn-primary btn-filled btn-medium arrow__hover--right-small undefined"
+              >
+                {t("menuLogOutText")}
+                <div className="ml-16">
+                  <ArrowIcon />
+                </div>
+              </button>
+            </Link>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
Where is it determined if a user is logged in or not?

Reply:
It is not. I added the GuardedApp to the .entry file. Seems like the way to do it.
Altho, we are still awaiting answers to how the login is going to flow in this context, so it will be revisited nonetheless. 
TODO added.

Reply:
The GuardedApp hoc is used in conjuction with a [guardedRequest](https://github.com/reload/dpl-react/blob/09bb7f547fcb520cd293c526151eddce8eab8d92/src/components/material/MaterialHeader.tsx#L51-L59).

The flow is:
* An anonymous tries to do something that triggers a request that requires the user to be authenticated
* The request is persisted by using redux persist
* The user is redirected to Adgangsplatformen for login
* After successful authentication the user is redirected back to where we started
* Because the app is wrapped in the guardedApp hoc the app is being held back until the persisted request has been fired
* When the request have been executed the app is loaded and the user can see the effects in the UI of the action that originally was taken

So that is a very long way of saying:
I do not think you need the guarded app hoc here, unless I don't get the full picture.

Reply:
Alright. Removed the GuardedApp. 
As this menu is accessible without having to reload the page, we need to check if the user i authorized.
I will visit this at a later stage when i know what this key will look like. Should be a pretty simple check to add at that point.